services-staticmap/src/main/java/com/mapbox/api/staticmap/v1/StaticMapCriteria.java

```diff
@@ -82,35 +82,35 @@
    *
    * @since 3.0.0
    */
-  public static final String SATELLITE_STREETS_STYLE = "satellite-streets-v10";
+  public static final String SATELLITE_STREETS_STYLE = "satellite-streets-v11";
 
   /**
    * Navigation specific style that shows only the necessary information while a user is driving.
    *
    * @since 3.0.0
    */
-  public static final String NAVIGATION_PREVIEW_DAY = "navigation-preview-day-v2";
+  public static final String NAVIGATION_PREVIEW_DAY = "navigation-preview-day-v3";
 
   /**
    * Navigation specific style that shows only the necessary information while a user is driving.
    *
    * @since 3.0.0
    */
-  public static final String NAVIGATION_PREVIEW_NIGHT = "navigation-preview-night-v2";
+  public static final String NAVIGATION_PREVIEW_NIGHT = "navigation-preview-night-v3";
 
   /**
    * Navigation specific style that shows only the necessary information while a user is driving.
    *
    * @since 3.0.0
    */
-  public static final String NAVIGATION_GUIDANCE_DAY = "navigation-guidance-day-v2";
+  public static final String NAVIGATION_GUIDANCE_DAY = "navigation-guidance-day-v3";
```

Comment:
Noting here that `v4` is available.

Reply:
Bumped in https://github.com/mapbox/mapbox-java/pull/1116